io.github.roseblume.rosemusic.yaml

```diff
@@ -0,0 +1,34 @@
+id: io.github.roseblume.rosemusic
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: rosary-music
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+modules:
+  - name: rose-music
+    buildsystem: simple
+    build-commands:
+      - ar -x *.deb
+      - tar -xf data.tar.gz
+      - install -Dm755 usr/bin/rosary-music /app/bin/rosary-music
+      - sed -i -e 's|Categories=|Categories=Education|g' usr/share/applications/Rosary-Music.desktop
+      - sed -i -e 's|Icon=rosary-music|Icon=io.github.roseblume.rosemusic|g' usr/share/applications/Rosary-Music.desktop
+      - install -Dm644 usr/share/applications/Rosary-Music.desktop /app/share/applications/io.github.roseblume.rosemusic.desktop
+      - install -Dm644 usr/share/icons/hicolor/128x128/apps/rosary-music.png /app/share/icons/hicolor/128x128/apps/io.github.roseblume.rosemusic.png
+      - install -Dm644 usr/share/icons/hicolor/32x32/apps/rosary-music.png /app/share/icons/hicolor/32x32/apps/io.github.roseblume.rosemusic.png
+      - install -Dm644 usr/share/icons/hicolor/256x256@2/apps/rosary-music.png /app/share/icons/hicolor/256x256@2/apps/io.github.roseblume.rosemusic.png
+      - install -Dm644 io.github.roseblume.rosemusic.metainfo.xml /app/share/metainfo/io.github.roseblume.rosemusic.metainfo.xml
+    sources:
+      - type: file
+        url: https://github.com/RoseBlume/Rose-Music/releases/download/v1.0.0/Rosary-Music_1.0.0_amd64.deb
+        sha256: f3b175139e6947f59db4f2c342c21fba2106df7a6056ce6bf5a05fa8634f8d56
+        dest-filename: rosemusic.deb
+        only-arches: [x86_64]
```

Comment:
so what happen in the aarch64 build? why not building from source to produce an aarch64 build?

Reply:
Unfortunately building from source is not possible without significantly increasing the applications size.

Reply:
I don't why the application size would change. But definitely building from source is preferred.

Reply:
I would have to switch to freedesktop and provide webkit2gtk

Reply:
I don't see the reason. Maybe you just don't need the GNOME in the end, which is a super set of the freedesktop SDK.

I reiterate. You should build from source.

Reply:
With Freedesktop  I would have to provide webkit2gtk. I have already seen this in practice and it results in a very large bundle.

Reply:
why would you have to use the freedesktop bundle? You still don't explain, just handwave.

Reply:
I would need both Rust and NodeJS in order to compile. In order to get them I would need to use the freedesktop sdk.

Reply:
That's not true at all.

Reply:
Where else can I get them?

Reply:
they are the exact same. There are many apps using the GNOME Sdk that use either or a combination of these SDK extensions. There is no difference. Just make sure they are installed.

Reply:
Alright thanks, I'll give it a shot.

Reply:
I got it to compile now I just have issues with committing the build.

Reply:
there are some infra issue in that regard. please be patient. We can retrigger a build later when issues are resolved.